include/networkit/io/NetworkitBinaryGraph.hpp

```diff
@@ -11,19 +11,21 @@ struct Header {
 	uint64_t offsetBaseData;
 	uint64_t offsetAdjLists;
 	uint64_t offsetAdjTranspose;
-	uint64_t offsetWeights;
+	uint64_t offsetWeightLists;
+	uint64_t offsetWeightTranspose;
 };
 
 enum WEIGHT_FORMAT {
-	VARINT = 0,
-	SIGNED_VARINT = 1,
-	DOUBLE = 2,
-	FLOAT = 3
+	none = 0,
+	unsignedFormat = 1,
+	signedFormat = 2,
+	doubleFormat = 3,
+	floatFormat = 4
```

Comment:
For consistency with the remainder of the file, we should keep the old ALL_UPPER_CASE names.